packages/spor-react/src/datepicker/Calendar.tsx

```diff
@@ -13,8 +13,15 @@ import { createTexts, useTranslation } from "../i18n";
 
 type CalendarProps = ReactAriaCalendarProps<DateValue> & {
   showYearNavigation?: boolean;
+  variant: ResponsiveValue<
+  "simple" 
+  | "with-trigger" 
```

Comment:
We probably shouldn't include the `simple` and `with-trigger` variants here – that's for the date input.